src/webview/WebViewAndroidBottomSheet.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Pressable, StyleSheet, Text, View } from 'react-native'
+import { Colors } from 'react-native/Libraries/NewAppScreen'
+import { WebViewEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import BottomSheet from 'src/components/BottomSheet'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { navigateToURI } from 'src/utils/linking'
+
+interface Props {
+  url: string
+  isVisible: boolean
+  onClose(): void
+  toggleBottomSheet(): void
+}
+
+export function WebViewAndroidBottomSheet({ url, isVisible, onClose, toggleBottomSheet }: Props) {
+  const { t } = useTranslation()
+  const openExternalLink = () => {
+    navigateToURI(url)
+    toggleBottomSheet()
+    ValoraAnalytics.track(WebViewEvents.webview_open_in_browser, { currentUrl: url })
+  }
+
+  return (
+    <BottomSheet
+      style={{ borderTopRightRadius: 0, borderTopLeftRadius: 0 }}
+      isVisible={isVisible}
+      onBackgroundPress={onClose}
+    >
+      <View>
+        <View style={styles.centerContainer}>
+          <Pressable
```

Comment:
Since it was Android only I was able to emulate this behavior with [android_ripple](https://reactnative.dev/docs/0.65/pressable#android_ripple-android). Can switch to a Touchable if the behavior is different, but in testing locally it looked the same.